samples/bluetooth/broadcast_audio_source/boards/nrf52833dk_nrf52833.overlay

```diff
@@ -0,0 +1,15 @@
+zephyr_udc0: &usbd {
+	compatible = "nordic,nrf-usbd";
+	status = "okay";
+
+	hs_0: hs_0 {
+		compatible = "usb-audio-hs";
+		mic-feature-mute;
+		mic-channel-l;
+		mic-channel-r;
+
+		hp-feature-mute;
+		hp-channel-l;
+		hp-channel-r;
+	};
```

Comment:
These exists in the `.dtsi` for the nRF 5340 ADK;

Why is it different for this board?

Reply:
My guess is that the Audio DK is considered to use USB for audio more than the standard DK.

Reply:
only CDC-ACM included by default in nRF SoC's with USB support, hence need an explicit overlay in this sample.